src/layouts/Media/components/MediaBreadcrumb.tsx

```diff
@@ -0,0 +1,67 @@
+import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from "@chakra-ui/react"
+import { BiChevronRight } from "react-icons/bi"
+import { useRouteMatch, Link as RouterLink } from "react-router-dom"
+
+import { deslugifyDirectory } from "utils"
+
+// NOTE: Media directories are recursively nested;
+// However, this nesting is marked by %2F in the url parameter.
+// Hence, we have to parse the mediaDirectoryName to get the nesting params.
+export const MediaBreadcrumbs = (): JSX.Element => {
+  const {
+    params: { mediaDirectoryName, siteName, mediaRoom: mediaType },
+  } = useRouteMatch<{
+    mediaDirectoryName: string
+    siteName: string
+    mediaRoom: "files" | "images"
+  }>()
+  const directories = mediaDirectoryName.split("%2F")
+  const firstItem = directories[0]
+
+  return (
+    <Breadcrumb spacing="2px" separator={<BiChevronRight color="text.body" />}>
+      {directories
+        .slice(1)
+        .reduce(
+          (acc, cur) => {
```

Comment:
Hey could we rename `acc` and `cur`? Am not too sure what acc and cur means :(

Reply:
yep oops this is on me - forgot to rename variables for clarity...